images/bootstrap/runner.sh

```diff
@@ -58,6 +58,12 @@ early_exit_handler() {
 # setup certificates before anything gets started
 setup_ca
 
+if [[ -n "$CONTAINER_HTTP_PROXY" && -n "$CONTAINER_HTTPS_PROXY" ]]; then
```

Comment:
nit: that check shouldn't be necessary, but IIRC we are setting both at the same time with the preset.

Reply:
I just don't want to set HTTP_PROXY and HTTPS_PROXY on every job -  want to make sure they get set on jobs that include the preset for the docker-mirror-proxy.